src/admin/class-datalayers.php

```diff
@@ -538,6 +635,76 @@ function remove_required($row) {
 		<?php
 	}
 
+	/**
+	 * Fetch the data from an external source URL.
+	 *
+	 * @param int|string $object_id The object ID.
+	 * @param bool       $single Whether to return a single result.
+	 *
+	 * @return array
+	 */
+	public static function fetch_datalayer_url_data( $object_id, $single = false ) {
+		if ( self::$datalayer_url_data ) {
+			return self::$datalayer_url_data;
+		}
+
+		$url      = get_post_meta( $object_id, 'datalayer_url', true );
+		$response = wp_remote_get( $url );
+
+		if ( is_wp_error( $response ) ) {
+			return [];
+		}
+
+		$body = wp_remote_retrieve_body( $response );
+		$data = json_decode( $body, true );
```

Comment:
Ook hier zou je geoPHP moeten gebruiken, anders ondersteun je alleen geoJson url's

Reply:
Heb het aangepast naar geoPHP::load met ook een try en catch om af te vangen of het bestand goed ingeladen kan worden